R/integrate_profile.R

```diff
@@ -189,9 +189,11 @@ integrate_profile <- function(x, alt_min, alt_max,
 integrate_profile.vp <- function(x, alt_min = 0, alt_max = Inf, alpha = NA,
                                  interval_max = 3600, interval_replace = NA, height_quantile = NA) {
   stopifnot(inherits(x, "vp"))
-  stopifnot(is.numeric(alt_min) | alt_min=="antenna")
-  stopifnot(is.numeric(alt_max))
-  stopifnot(is.na(alpha) || is.numeric(alpha))
+  assertthat::assert_that(
+    is.numeric(alt_min) || alt_min=="antenna"
+  )
+  assertthat::assert_that(assertthat::is.number(alt_max))
+  assertthat::assert_that(is.na(alpha) || is.numeric(alpha))
```

Comment:
`is.numeric` should be `is.number`, alpha cannot be a vector

Reply:
If I make this change, the following tests fail: 
```R
expect_s3_class(integrate_profile(c(example_vp,example_vp)), "vpi")
```

https://github.com/adokter/bioRad/blob/c87ec9d5caa23a4789c56408a9aa2537193959dd/tests/testthat/test-integrate_profile.R#L3

And 

```R
expect_error(
    integrate_profile(example_vp, alt_min = 500, alt_max = 300),
    regexp = "'alt_min' should be smaller than 'alt_max'",
    fixed = TRUE
    )
```

Can you see what is going wrong here?

Reply:
Found it, need to allow NA